acis_thermal_check/test_utils.py

```diff
@@ -10,13 +9,18 @@
 test_data_dir = "/data/acis/thermal_model_tests"
 
 class TestOpts(object):
-    def __init__(self, short_msid, run_start, outdir, oflsdir=None, 
-                 days=21.0, ccd_count=6, fep_count=6, vid_board=1,
-                 clocking=1, simpos=75616.0, pitch=150.0, T_init=None,
-                 dh_heater=0, cmd_states_db='sybase'):
+    def __init__(self, short_msid, run_start, outdir, model_spec=None,
+                 load_week=None, days=21.0, ccd_count=6, fep_count=6,
+                 vid_board=1, clocking=1, simpos=75616.0, pitch=150.0,
+                 T_init=None, dh_heater=0, cmd_states_db='sybase'):
+        self.load_week = load_week
         self.run_start = run_start
         self.outdir = outdir
-        self.oflsdir = oflsdir
+        if load_week is None:
+            self.oflsdir = None
+        else:
+            load_year = "20%s" % load_week[-2:]
+            self.oflsdir = "/data/acis/LoadReviews/%s/%s/ofls" % (load_year, load_week)
```

Comment:
For now I think it's safe to assume we'd either be doing tests out of flight data directories or using completely specified initial conditions. When we do test LR runs we put them in the same directory structure with a different name schema for `load_week`, so this would be compatible with those as well.